docs/content/index.md

```diff
@@ -46,67 +46,67 @@ title: cssnext
   <div class="r-Grid-cell r-minM--2of3">
   <h2 class="cssnext-Section-title"><a href="/features/">Features</a></h2>
     <ul class="r-Grid cssnext-List cssnext-FeaturesList">
-      <li class="r-Grid-cell r-all--1of2">
+      <li class="r-Grid-cell r-all--1of2 r-maxS--2of2">
```

Comment:
Hum, `2of2` seems to be 100%, which is the default value right ? Maybe we should replace `r-all--1of2` by `r-minL--1of2` instead (or minM) ?